src/client/components/loginPage/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+
+export default class LoginPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            username: ''
+        }
+
+        this.login = this.login.bind(this);
+        this.logout = this.logout.bind(this);
+    }
+
+    async login() {
+        const loginData = await (await fetch('http://localhost:5000/login', { method: 'POST'})).json();
```

Comment:
Would be best to not hard code this API  URLs. We could use [Webpack's DefinePlugin](https://webpack.js.org/plugins/define-plugin/) here, but I'll be curious what @gnarf thinks as well.